core/group/service.go

```diff
@@ -335,6 +337,13 @@ func (s Service) RemoveUsers(ctx context.Context, groupID string, userIDs []stri
 			err = errors.Join(err, currentErr)
 		}
 	}
+
+	if err == nil {
+		audit.GetAuditor(ctx, schema.PlatformOrgID.String()).LogWithAttrs(audit.GroupMemberRemovedEvent, audit.GroupTarget(groupID), map[string]string{
+			"userIDs": strings.Join(userIDs, ", "),
```

Comment:
should we sort these ids to make it deterministic?

Reply:
I thought of emitting events per user, to avoid losing any events in case of failure in any one user removal or infra crashes. Can you recheck once?